package/tests/testCPU.py

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env python
+#
+# Probably we are python2, but use python3 syntax as much as possible
+from __future__ import print_function, unicode_literals
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+import unittest
+
+def setupConfigurableTest(threads=1, procs=1, time=10, slack=0.75):
+    '''Wrap the class definition in a function to allow arguments to be passed'''
+    class configurableProcessMonitor(unittest.TestCase):
+        def test_runTestWithParams(self):
+            burn_cmd = ['./burner', '--time', str(time)]
+            if procs != 1:
+                burn_cmd.extend(['--procs', str(procs)])
+            if threads != 1:
+                burn_cmd.extend(['--threads', str(threads)])
+            burn_p = subprocess.Popen(burn_cmd, shell = False)
+    
+            prmon_cmd = ['../prmon', '--pid', str(burn_p.pid)]
+            prmon_p = subprocess.Popen(prmon_cmd, shell = False)
+    
+            burn_rc = burn_p.wait()
+            prmon_rc = prmon_p.wait()
+    
+            self.assertEqual(prmon_rc, 0, "Non-zero return code from prmon")
+            prmonJSON = json.load(open("prmon.json"))
+            totCPU = prmonJSON["Max"]["totUTIME"] + prmonJSON["Max"]["totSTIME"]
+            self.assertLess(totCPU, time*threads*procs, "Too high value for CPU time")
+            self.assertGreater(totCPU, time*threads*procs*slack, "Too low value for CPU time")
+    
```

Comment:
I think somewhere around here it would be nice to have a simple printout of the values `totCPU` and `time*threads*procs` w/ and w/o `*slack` such that the user will have a bit more information (might be useful especially if the test fails).

Reply:
The failure message should be more informative now (of course, all output is hidden at the `make test` stage, but it's in `Testing/Temporary/LastTest` and if run "by hand" you see it)